lib/auth0/api/v2/users.rb

```diff
@@ -27,9 +30,9 @@ def users(options = {})
             connection:     options.fetch(:connection, nil),
             fields:         options.fetch(:fields, nil),
             include_fields: options.fetch(:include_fields, nil),
-            q:              options.fetch(:q, nil)
+            q:              options.fetch(:q, nil),
```

Comment:
Nothing you need to worry about now since the convention is used, but just using as an opportunity to share info:

`.fetch` is typically used on a hash when you want to look for a key and raise an exception if it is not empty, or return a specific default. Accessing a hash key via the `[]` method is equivalent to `.fetch(key, :nil)`, so when `nil` is the desired default return value, you'll typically find the `[]` method is used.

Reply:
@machuga - So something like:

```ruby
request_params = {
  ...
  q: [:q]
  ...
}
```

?

Reply:
Close: 
```ruby
options[:key] === options.fetch(:key, nil)
```